File: app/src/main/java/com/example/msimple/data/vos/SessionVO.java
package com.example.msimple.data.vos;

public class SessionVO {

    private String session_id;
    private String title;
    private int length_in_seconds;
    private String file_path;

    public String getSession_id() {
        return session_id;
    }

    public String getTitle() {
        return title;
    }

    public String getTime() {
        return (getLength_in_seconds()/60)+" mins";
    }

    public String getFile_path() {
        return file_path;
    }

    public int getLength_in_seconds() {
        return length_in_seconds;
    }
}

File: app/src/main/java/com/example/msimple/data/vos/SessionVO.kt
package com.example.msimple.data.vos

class SessionVO {

    val session_id: String? = null
    val title: String? = null
    val length_in_seconds: Int = 0
    val file_path: String? = null

    val time: String
        get() = (length_in_seconds / 60).toString() + " mins"
}
